os/se/ameba/security_ameba_wrapper_tz.c

```diff
@@ -201,16 +201,18 @@ int se_ameba_hal_deinit(void)
 int se_ameba_hal_free_data(hal_data *data)
 {
 	AWRAP_ENTER;
-	int ret = HAL_SUCCESS;
 
-	up_allocate_secure_context(CONFIG_SE_SECURE_CONTEXT_SIZE);
-	ret = ameba_hal_free_data(data);
```

Comment:
Then, who calls this?

Reply:
Is that moved into cmse_implib.a?

Reply:
Currently no one need to use this, this will pass data pointer into secure to free it.
But it is not needed, because in non-secure there is no secure pointer.
So it have be be remove and replace with normal free.
Previously it did not cause problem due to no one is using it.